tools/pubsys/src/aws/publish_ami/mod.rs

```diff
@@ -574,7 +584,7 @@ mod error {
         },
 
         #[snafu(display(
-            "Failed to modify permissions of {} of {} snapshots",
+            "No AMIs were made public because we failed to modify permissions of {} of {} snapshots",
             error_count, error_count + success_count,
         ))]
         ModifySnapshotAttributes {
```

Comment:
Non-blocking comment/nit:

I understand we need to tell users that no AMIs were made public when this error occurs, but can we do it by wrapping the original error message so that `ModifySnapshotAttributes` stands on its own and can be used for errors that occur outside of the context of making AMIs public?

Same comment for `WaitAmi` in the context of publishing a single AMI amongst many.

Reply:
That's not easy to do because `ModifyImagesAttributes` is a case where some of the AMIs are published and some aren't. I see your point though, because these functions are `pub(crate)` and used by the `ami` command.

I will need to find a way to do this.